index.html

```diff
@@ -1819,6 +1819,50 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
     The <dfn>NFCWriter</dfn> is an object used for writing data to NFC devices
     such as tags.
   </p>
+  <p>
+    A {{NFCWriter}} object has the following <a data-cite=
+    "ECMASCRIPT#sec-object-internal-methods-and-internal-slots">
+    internal slots</a>:
+  </p>
+  <table class="simple">
+    <thead>
+     <tr>
+      <th>Internal Slot</th>
+      <th>Description (<em>non-normative</em>)</th>
+     </tr>
+    </thead>
+    <tbody data-link-for="NFCPushOptions">
+     <tr>
+      <td><dfn>[[\PushOptions]]</dfn></td>
+      <td>
+        This slot holds the provided {{NFCPushOptions}} value.
```

Comment:
why not follow the style used elsewhere?


Internal slot | Initial value | Description (non-normative)
[[PayloadData]] | Empty byte sequence. | A byte sequence representing the whole or a subset of the PAYLOAD field data.

Reply:
[[\PushOptions]] | `null` | The {{NFCPushOptions}} value for a pending push

"This slots hold..." seems unneeded

Reply:
I copied it from NFCReader... so it follows the style used there. So we should then change both?

Reply:
Yeah I see now we have two styles, would be good to fix